maps/map_files/Ice_Colony_v3/Shivas_Snowball.dmm

```diff
@@ -15887,6 +15887,12 @@
 	},
 /turf/open/auto_turf/snow/layer1,
 /area/shiva/exterior/cp_colony_grounds)
+"kCQ" = (
```

Comment:
- Same as before, use Noop turf/area when there are no changes between a small area on the nightmare insert and base map.
- Why are the walls changing color? Did the CLF decide to build new grey walls on the outside near the offices?
- There is a HPR still inside of this office, did the CLF not search around or use it etc.?
- ![bilde](https://user-images.githubusercontent.com/59782240/184452718-adc98676-8b9f-467e-9eb3-2a6bf2f745a6.png)
- Too many posters again, touch it down a little.
- Damaged barricades needs their health VV'd as well.
- Calm down on the landmarks for spawners, this isn't the Cryocells on-board the USS Almayer, delete a fair few, consider locations where survivors would "wake" up or be standing guard etc.
- Calm down on the ammo for the CLF here as well, remember, they have been fighting and are already underfunded etc.
- ![bilde](https://user-images.githubusercontent.com/59782240/184452992-6eeef8a6-5d73-4255-bdd2-6bed92e0957e.png)